core/workspace_comment/workspace_comment_svg.js

```diff
@@ -69,6 +69,17 @@ Blockly.WorkspaceCommentSvg = function(workspace, content, height, width,
    */
   this.width_ = width;
 
+  /** @type {boolean} */
+  this.rendered = false;
```

Comment:
Let's make rendered private (rendered_).  I think it's public in blocks only for legacy reasons.